source/reference/minio-mc.rst

```diff
@@ -151,38 +151,20 @@ The following table lists :mc-cmd:`mc` commands:
    * - Command
      - Description
 
```

Comment:
Since this is a quickref, i'd argue to leave this list as-is as a "index"

If it feels too heavy handed I'd rather break the table up into H2s - e.g.

Object Operations
~~~~~~~~~~~~~~~

Bucket Management
~~~~~~~~~~~~~~~~~

Replication
~~~~~~~~~

Identity and Access Management
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

etc. Thoughts?